cdap-docs/build.sh

```diff
@@ -348,6 +348,20 @@ function build_docs_package() {
       echo "Could not create zipped doc set ${TARGET_PATH}/${PROJECT_VERSION}"
       return ${errors}   
   fi
+  echo "Moving zip ${zip_dir_name}"
+  mv ${zip_dir_name}.zip ${TARGET_PATH}/${PROJECT_VERSION}
```

Comment:
For now, let's make this a copy, rather than a move. Once we get rid of the older deployment and have verified that the new method works for older releases, then we can change this. For now, I'll simply remove the ZIP in the top-level `target` in the build, before artifacts are published.